tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py

```diff
@@ -331,6 +331,22 @@ def default_test(self, tables_and_test_values):
                     fields = [table_info.get('fields', [])[0]]
                     self.assertRowIDsMatch(fq_sandbox_name, fields, values)
 
+        def copy_vocab_tables(self, vocabulary_id):
+            """
+            A function for copying the vocab tables to the test dataset_id
+            :param vocabulary_id:
+            :return:
+            """
+            # Copy vocab tables over to the test dataset
+            vocabulary_dataset = self.client.get_dataset(vocabulary_id)
+            for src_table in self.client.list_tables(vocabulary_dataset):
+                schema = self.client.get_table_schema(src_table.table_id)
+                destination = f'{self.project_id}.{self.dataset_id}.{src_table.table_id}'
+                dst_table = self.client.create_table(bigquery.Table(
+                    destination, schema=schema),
+                                                     exists_ok=True)
+                self.client.copy_table(src_table, dst_table)
```

Comment:
I also see some BigQuery errors saying `Already Exists: Table test_project:test_rdr.concept` etc. for all vocabulary tables. Is this caused by `copy_table`? Doesn't seem to have any other impact.

Reply:
This is a good catch. There are some other rules that are using a function named `copy_vocab_tables`.  They are using a smaller subset of the vocabulary tables and were not using `exists_ok=True`.  I updated them to all use the base class method and removed their independent definitions of the method.  I also made sure they were all adding the vocabulary tables to their list of tables, so they are cleaned up.